Formula/chrome-cli.rb

```diff
@@ -37,7 +37,14 @@ def install
     inreplace "chrome-cli/App.m", "com.google.Chrome.canary", "org.Chromium.chromium"
     xcodebuild "SDKROOT=", "SYMROOT=build"
     bin.install "build/Release/chrome-cli" => "chromium-cli"
-  end
+
+    # Brave builds; see:
+    # https://github.com/prasmussen/chrome-cli/issues/57
+    rm_rf "build"
+    inreplace "chrome-cli/App.m", "org.Chromium.chromium", "com.brave.Browser"
+    xcodebuild "SDKROOT=", "SYMROOT=build"
+    bin.install "build/Release/chrome-cli" => "brave-cli"
+ end
```

Comment:
This needs to be enabled upstream, we're not going to randomly patch browser names into some source.

Reply:
What do you mean **enabled upstream**? 

Just followed the same pattern as for Canary (see line 30) and Chromium (see line 37) as the app does not support and the `chrome-cli` maintainer is not doing any fixes on it.

Reply:
There's a big difference between "patching in support for a beta release of the same code" and "patching in support for another browser" IMO. If this is something the author is fine with they can use a buildsystem that allows for this configuration. If they are not, we should not do this.

Reply:
Fair enough. I will close PR then, hopefully, the author will add such support but I feel he abandoned the project years ago.